src/Filter/DateRangeFilter.php

```diff
@@ -33,6 +31,7 @@ class DateRangeFilter extends AbstractDateFilter
 
     public function getFieldType()
     {
-        return $this->getOption('field_type', DateRangeType::class);
+        // NEXT_MAJOR: Import the class from "sonata-project/form-extensions" and use `DateRangeType::class` instead.
+        return $this->getOption('field_type', 'Sonata\Form\Type\DateRangeType');
```

Comment:
Wouldn't it break, if you remove the core bundle depdency?

Reply:
I think you're (sadly) right, since "Sonata\Form\Type\DateRangeType" is used as default value if the option "field_type" doesn't exist.
This means I should add one more item to the list of things I must be aware of:
* Usage of classes (new instances, static method calls, argument and return declarations, inheritance, `::class`, etc)
* Services
* Twig filters
* **Form types**

And finally, this also means this PR can't be merged :sob: 

Thank you!

Reply:
> And finally, this also means this PR can't be merged sob

That's sadly right :( We will have a hard upgrade path when releasing the new major versions